django/forms/widgets.py

```
@@ -377,7 +377,14 @@ def is_initial(self, value):
         """
         Return whether value is considered to be initial value.
         """
-        return bool(value and hasattr(value, 'url'))
+        if six.PY2:
```

Could you add a comment `# hasattr() masks exceptions on Python 2`